src/connect/ConnectWizard.ts

```diff
@@ -199,7 +199,7 @@ export class ConnectWizard {
             totalSteps: this.NumberOfSteps,
             placeholder: `Choose a service to redirect to your machine`,
             items: serviceChoices.sort((s1, s2) => s1.label < s2.label ? -1 : 1),
-            activeItem: serviceChoices[0]
+            activeItem: null
```

Comment:
I'm wondering if we could report this issue on the VS Code side and let them fix it? I mean, even if we mitigate it here for our extension, this will still impact all of the customers using other extensions relying on QuickPicks.
Also, removing the activeItem means that we make the experience slightly worse for our customers, whereas a global VS Code fix would make everyone happy.
Sorry for the side comment, hope it doesn't sidetrack this PR. As QuickPicks are a common VS Code component it would make sense to me that the fix is global to VS Code.

Reply:
Hiya Dan, Thank you for this, 🙏 ❤️  yeah I felt the same but I wanted to make sure I did homeowrk because for some genuine vscode bug I have raised one with them alread.

In this specific instance: I did tried this, and it si not vscode bug, so eswswentially if you see vanilla quickpick list - in vscode implementation of even `ctrl + shift + P` they donot select first time if they do that means no `placeholder text`

some of the old bugs they have fixed for placeholder to read are as follows:

* https://github.com/Microsoft/vscode/issues/51824
* https://github.com/microsoft/vscode/issues/57501
* https://github.com/microsoft/vscode/issues/51832

So esentially if non selected it reads it if not then its a desired behavior for screen reading.

Reply:
☕️❤️ Closing PR and moving this as a bug against the vscode is the best option. I donot have vanilla reproducible steps, so for discussions I will fyi folks within this conversation for that workitem.